tpcx_bb/queries/q15/tpcx_bb_query_15_sql.py

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+# Copyright (c) 2019-2020, BlazingSQL, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+
+
+from blazingsql import BlazingContext
+from xbb_tools.cluster_startup import attach_to_cluster
+from dask_cuda import LocalCUDACluster
+from dask.distributed import Client
+import os
+
+from xbb_tools.utils import (
+    benchmark,
+    tpcxbb_argparser,
+    run_bsql_query,
+)
+
+cli_args = tpcxbb_argparser()
+
+
+@benchmark(
+    compute_result=cli_args["get_read_time"], dask_profile=cli_args["dask_profile"]
+)
+def read_tables(data_dir):
+    bc.create_table("store_sales", data_dir + "/store_sales/*.parquet")
+    bc.create_table("date_dim", data_dir + "/date_dim/*.parquet")
+    bc.create_table("item", data_dir + "/item/*.parquet")
+
+
+@benchmark(dask_profile=cli_args["dask_profile"])
+def main(data_dir, client):
+    read_tables(data_dir)
+
+    query = """
+        SELECT *
+        FROM 
+        (
+            SELECT
+                cat,
+                ( (count(x) * SUM(xy) - SUM(x) * SUM(y)) / (count(x) * SUM(xx) - SUM(x) * SUM(x)) )  AS slope,
+                (SUM(y) - ((count(x) * SUM(xy) - SUM(x) * SUM(y)) / (count(x) * SUM(xx) - SUM(x)*SUM(x)) ) * SUM(x)) / count(x) AS intercept
+            FROM 
+            (
+                SELECT
+                    i.i_category_id AS cat, 
+                    s.ss_sold_date_sk AS x,
+                    CAST(SUM(s.ss_net_paid) AS DOUBLE) AS y,
+                    CAST(s.ss_sold_date_sk * SUM(s.ss_net_paid) AS DOUBLE) AS xy,
+                    CAST(s.ss_sold_date_sk * s.ss_sold_date_sk AS DOUBLE) AS xx
+                FROM store_sales s
+                INNER JOIN item i ON s.ss_item_sk = i.i_item_sk
+                INNER JOIN date_dim d ON s.ss_sold_date_sk = d.d_date_sk
+                WHERE s.ss_store_sk = 10
```

Comment:
Would be nice to factor out the `10` into a variable with a name matching the original Spark for clarity. You can probably use the ones from the pure Dask query

Reply:
Additional to the  `10` value (`q15_store_sk`),  the variables `q15_startDate` and  `q15_endDate` also were considered as original Spark version.